src/main/java/org/terasology/launcher/packages/PackageManager.java

```diff
@@ -31,9 +39,16 @@
 public class PackageManager {
 
     private static final Logger logger = LoggerFactory.getLogger(PackageManager.class);
+    private static final String INSTALL_DIRECTORY = "games";
```

Comment:
I think this should also come from some configuration object that is  passed to the package manager.